terraform/main.tf

```diff
@@ -204,3 +207,13 @@ module "batch_job_queues" {
   aurora_max_vcpus = local.aurora_max_vcpus
   gcs_secret       = data.terraform_remote_state.core.outputs.secrets_read-gfw-gee-export_arn
 }
+
+module "api_gateway" {
```

Comment:
I see you delete the API gateway code here, but I can't find where this module is defined in either of your PRs? Was it already defined?

Reply:
we had gitignore rule for the whole terraform folder:) it's pushed now - thanks for catching that